res/values/strings.xml

```diff
@@ -1439,68 +1426,68 @@
     <string name="preferences_advanced__always_relay_calls">Always relay calls</string>
     <string name="preferences_app_protection__app_access">App access</string>
     <string name="preferences_app_protection__communication">Communication</string>
     <string name="preferences_chats__chats">Chats</string>
     <string name="preferences_notifications__messages">Messages</string>
     <string name="preferences_notifications__events">Events</string>
-    <string name="preferences_notifications__in_chat_sounds">In-chat sounds</string>
+    <string name="preferences_notifications__in_chat_sounds">Also sound if Signal is open already</string>
     <string name="preferences_notifications__show">Show</string>
     <string name="preferences_notifications__calls">Calls</string>
     <string name="preferences_notifications__ringtone">Ringtone</string>
     <string name="preferences_chats__show_invitation_prompts">Show invitation prompts</string>
-    <string name="preferences_chats__display_invitation_prompts_for_contacts_without_signal">Display invitation prompts for contacts without Signal</string>
+    <string name="preferences_chats__display_invitation_prompts_for_contacts_without_signal">Display a suggestion to invite your contact to Signal, above an SMS chat with a contact who is not on Signal.</string>
     <string name="preferences_chats__message_text_size">Message font size</string>
-    <string name="preferences_events__contact_joined_signal">Contact joined Signal</string>
-    <string name="preferences_notifications__priority">Priority</string>
+    <string name="preferences_events__contact_joined_signal">Contact now uses Signal</string>
+    <string name="preferences_notifications__priority">Urgency</string>
     <string name="preferences_communication__category_sealed_sender">Sealed Sender</string>
-    <string name="preferences_communication__sealed_sender_display_indicators">Display indicators</string>
+    <string name="preferences_communication__sealed_sender_display_indicators">Show sealed sender indicator</string>
     <string name="preferences_communication__sealed_sender_display_indicators_description">Show a status icon when you select "Message details" on messages that were delivered using sealed sender.</string>
     <string name="preferences_communication__sealed_sender_allow_from_anyone">Allow from anyone</string>
-    <string name="preferences_communication__sealed_sender_allow_from_anyone_description">Enable sealed sender for incoming messages from non-contacts and people with whom you have not shared your profile.</string>
+    <string name="preferences_communication__sealed_sender_allow_from_anyone_description">Enable sealed sender for incoming messages from non-contacts and people to whom you have not made your profile visible. This comes at the increased risk of receiving unwanted messages.</string>
```

Comment:
I want to make clear that this is to be considered an unintentional but unavoidable side effect rather than an intentional effect. "it comes at the cost of" shows that this is a trade-off rather than an intent.
I can change "This comes at the increased risk of receiving..." to "this comes at the cost of an increased risk of receiving..." if that helps. Otherwise I recommend to keep it this way.

Reply:
Ok, keep your original text.